main_column.py

```diff
@@ -1,21 +1,31 @@
 """Distillation column model for 2018 PSE conference"""
```

Comment:
Add a correct reference to PSE paper

Reply:
added the reference by Ghouse et al., 2018.